hack/e2e.sh

```diff
@@ -52,7 +52,10 @@ export VSPHERE_USERNAME="${GOVC_USERNAME}"
 export VSPHERE_PASSWORD="${GOVC_PASSWORD}"
 export VSPHERE_SSH_AUTHORIZED_KEY="${VM_SSH_PUB_KEY}"
 export VSPHERE_SSH_PRIVATE_KEY="/root/ssh/.private-key/private-key"
-export E2E_CONF_FILE="${REPO_ROOT}/test/e2e/config/vsphere-ci.yaml"
+export E2E_CONF_FILE="${REPO_ROOT}/test/e2e/config/vsphere.yaml"
+export E2E_CONF_OVERRIDES=""
```

Comment:
```suggestion
export E2E_CONF_OVERRIDES_FILE=""
```
for consistency with E2E_CONF_FILE